backend/fms_core/resources/sample.py

```diff
@@ -175,7 +175,25 @@ def import_obj(self, obj, data, dry_run):
         except Exception as e:
             individual_created = False
             individual = None
-            errors["individual"] = ValidationError(e.messages.pop(), code="invalid")
+            errors["individual"] = ValidationError("Individual could not be processed due to missing or invalid information.", code="invalid")
+
+        #Individual field errors
+        validate_specific_field(errors, "Taxon", "taxon", data)
```

Comment:
would there be a less repetitive way of doing it? :)

Reply:
`#Individual field errors
        fields = {
            "Taxon": "taxon",
            "Individual ID": "individual_id",
            "Container Kind":"container_kind",
            "Container Name": "container_name",
            "Container Barcode": "container_barcode"
        }
        
        validate_specific_fields(data, errors, fields)`

Then function will traverse the dictionary. What do you think?